tests/switch/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -169,7 +169,7 @@ endswitch;
 =====================================output=====================================
 <?php
 switch (1) {
-    default:;
+    default:
```

Comment:
Not sure about this one - both seems to be valid PHP code.

Reply:
It is expected, because here noop node and now we don't print them